.editorconfig

```diff
@@ -13,13 +13,22 @@ indent_size = 2
 ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
 ij_kotlin_packages_to_use_import_on_demand = java.util.*, io.ktor.**
 
+ktlint_code_style = intellij_idea
+ktlint_standard_annotation = disabled
+ktlint_standard_class-naming = disabled
+ktlint_standard_class-signature = disabled
+ktlint_standard_comment-wrapping = disabled
+ktlint_standard_filename = disabled
+ktlint_standard_function-expression-body = disabled
+ktlint_standard_function-signature = disabled
+ktlint_standard_if-else-bracing = enabled
+ktlint_standard_if-else-wrapping = enabled
```

Comment:
I don't care for the if-else rules 👎

Reply:
The only reason why I've enabled this rule is that we've had only 19 violations of it. So it seems reasonable to enable it as we aligned to it anyway.